front/package.json

```diff
@@ -10,6 +10,7 @@
     "@nivo/line": "^0.80.0",
     "@nivo/tooltip": "^0.80.0",
     "@openapi-contrib/openapi-schema-to-json-schema": "^5.1.0",
+    "@osrd-project/netzgrafik-frontend": "0.0.0-snapshot.23b18bb8c907e3d6d88637617df86829eef7d5b9",
```

Comment:
Just for review purposes, this will be changed to point to the official NPM SBB organization, at soon as they release their next release